Engines/Wine/Verbs/dotnet20/script.js

```diff
@@ -1,43 +1,57 @@
 include(["engines", "wine", "engine", "object"]);
 include(["engines", "wine", "plugins", "override_dll"]);
 include(["utils", "functions", "net", "resource"]);
-include(["engines", "wine", "verbs", "luna"]);
 include(["utils", "functions", "filesystem", "files"]);
 include(["engines", "wine", "plugins", "windows_version"]);
+include(["engines", "wine", "verbs", "removemono"]);
 
 /**
 * Verb to install .NET 2.0
 * @returns {Wine} Wine object
 */
 Wine.prototype.dotnet20 = function () {
+    var osVersion = this.windowsVersion();
+    if (osVersion == null)
```

Comment:
I would recommend to add a `{` and `}` around the if-then body. This makes it easier to read and prevents errors.

Reply:
If Codacy doesn't complain about this, we should configure it to do so.